### PR TITLE
Create qst.yml for #574

### PR DESCRIPTION
**Pull request by ryangurn**
Contributing on behalf of @bobb34 for #574

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Comment by bobb34** [comment]
I am sorry, but i do not understand how to put the x's into the boxes?

**Comment by bobb34** [comment]
On 2024-04-23 21:00, Ryan Gurnick wrote:

> Contributing on behalf of @bobb34 [1] for #574 [2]
> 
> Thanks for taking the time to suggest an addition to 
> awesome-selfhosted!
> 
> To ensure your Pull Request is dealt with swiftly, please check the 
> following (check the boxes [x]):
> 
> * x Submit one item per pull request. This eases reviewing and speeds 
> up inclusion.
> * x You have searched the repository for any relevant issues [3] or PRs 
> [4], including closed ones.
> * x Any software you are adding is not already listed at any of 
> awesome-sysadmin [5], staticgen.com [6], staticsitegenerators.net [7], 
> dbdb.io [8].
> * x The file you are adding is formatted as described in addition.md 
> [9].
> * x Demo links should only be used for interactive demos, i.e. not 
> video demonstrations.
> * x Comments and unused optional fields have been removed.
> * x The file you are adding uses kebab-case [10] file naming, for 
> example my-awesome-software.yml.
> * x Values for platform are the main server-side requirements for the 
> software. Don't include frameworks or specific dialects.
> * x Any software project you are adding to the list is actively 
> maintained.
> * x Any software project you are adding was first released more than 4 
> months ago.
> * x Any software project you are adding has working installation 
> instructions.
> * x You understand that your Pull Request will be merged ~1 week after 
> approval, to allow for further comments if needed.
> 
> -------------------------
> 
> YOU CAN VIEW, COMMENT ON, OR MERGE THIS PULL REQUEST ONLINE AT:
> 
> https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/674
> 
> COMMIT SUMMARY
> 
> * b032812 [11] Create qst.yml
> 
> FILE CHANGES
> 
> (1 file [12])
> 
> * A software/qst.yml [13] (13)
> 
> PATCH LINKS:
> 
> * 
> https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/674.patch
> * 
> https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/674.diff
> 
> --
> Reply to this email directly, view it on GitHub [14], or unsubscribe 
> [15].
> You are receiving this because you were mentioned.Message ID: 
> ***@***.***>



Links:
------
[1] https://github.com/bobb34
[2] 
https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues/574
[3] https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues
[4] https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls
[5] https://github.com/n1trux/awesome-sysadmin
[6] https://www.staticgen.com/
[7] https://staticsitegenerators.net/
[8] https://dbdb.io/browse
[9] 
https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md
[10] https://en.wikipedia.org/wiki/Letter_case#Kebab_case
[11] 
https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/674/commits/b032812fe2f7ebda930b12b37917c6b66a4800be
[12] 
https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/674/files
[13] 
https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/674/files#diff-479cb4434ec64840a8f027055e95c1b41f90c4921033dd5fb16a5a5c311763a7
[14] 
https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/674
[15]

**Comment by ryangurn** [comment]
Hopefully that change to remove the double quotes addresses the error.

**Review by Ki-er — CHANGES_REQUESTED** [review]
This error was because there was a ' ' after the '.' Please add the "" back to the file.

**Comment by ryangurn** [comment]
@Ki-er Thanks for clarifying that!

File: software/qst.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+source_code_url: "https://sourceforge.net/projects/qstonline/"
+description: "QST, the worlds unparalleled open source online assessment software. From a quick quiz on your phone to large scale, high stakes, proctored desktop testing, we make it easy, secure and economical."
+licenses:
+  - GPL-2.0
+platforms:
+  - Perl
+tags:
+  - Learning and Courses
+  - Knowledge Management Tools
+depends_3rdparty: false
```
Comment (kokomo123) [inline software/qst.yml:12]:
```suggestion
```
As it's false by default, feel free to omit this line. Other then that, looks all good to me!

**Review by kokomo123 — APPROVED** [review]
LGTM, Thanks!

File: software/qst.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: "QST"
+website_url: "https://qstonline.org"
+source_code_url: "https://sourceforge.net/projects/qstonline/"
+description: "QST, the worlds unparalleled open source online assessment software. From a quick quiz on your phone to large scale, high stakes, proctored desktop testing, we make it easy, secure and economical."
```
Comment (nodiscc) [inline software/qst.yml:4]:
```suggestion
description: "Online assessment software. From a quick quiz on your phone to large scale, high stakes, proctored desktop testing, easy, secure and economical."
```

Simple/more neutral description.

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/qst.yml (hunk 2 above)
Comment (nodiscc) [reply software/qst.yml:4]: @ryangurn ping

**Review by nodiscc — APPROVED** [review]

**Pull request merged**